docs/user-guide/faq.md

```diff
@@ -87,3 +87,11 @@ keymaps = [
 
 [`map`]: https://nixos.org/manual/nix/stable/language/builtins#builtins-map
 [`keymaps`]: ../keymaps
+
+## How do I avoid using `nix` to install dependent packages
```

Comment:
Maybe this title should be more focused on the problem than the solution? Not sure...

Something in the spirit of "How to use system-provided binaries instead of packages installed by nixvim" but less verbose and overly technical?

I'm happy with the current title if we can't think of anything immediately though.